test_AllocateToAdOps_Invocable.cls

```diff
@@ -17,10 +17,9 @@ newOpp.SendingToAdOps__c = false;
 list<Opportunity> testOpp_l = [select id, processed_by_lookup__c from opportunity where id = :newOpp.id];
```

Comment:
Variables should use the `lowerCamelCase` naming convention. Please rename accordingly.